src/Feldspar/Onnx/Operators.hs

```diff
@@ -259,8 +261,8 @@ onnxBatchNormalization attrs xs gamma beta mean var = ys
         epsilon = value $ P.realToFrac $ getAttr attrs aaFloat 1e-5 "epsilon"
 
 -- | Flatten a tensor to a matrix
-onnxFlatten :: Pushy vec => Attrs -> vec a -> Push DIM2 a
-onnxFlatten attrs vec = flatPush (P.fromIntegral $ getAttr attrs aaInt 1 "axis") $ toPush vec
+onnxFlatten :: (Pushy vec, Syntax a) => Attrs -> vec a -> Pull DIM2 a
```

Comment:
Is `Syntax` a way to get `Storable`?

Reply:
'Syntactic a' is necessary; the definition of 'store' from the class declaration is:
  store :: Syntax a => vec a -> Manifest (VecShape vec) a